### DISCUSSION TITLE
Search for values in form

### DISCUSSION THREAD

**Discussion by BTreeTech** [opengisch/QField#4477] [Q&A]
I have value map widgets with various lists like tree species, I would like to be able to allow users to type a few letters to find a value rather than browsing. I have seen a video showing this behaviour with a magnifying glass but can't get it to work. Not sure if I am using the wrong widget or if they are not configured correctly

**Comment by VxTedxV** [comment]
In QGis>layer properties>attribute form,
available with "value relation" widget and tick "use completer" checkbox

**Reply by BTreeTech** [reply]
Thanks, the value relation widget is much better, sadly even when I have "use completer" ticked the search doesn't show up in Qfield - it does in QGIS but not on either of my Qfield testing devices.
After some more testing it does work but only if multiple selections isn't ticked

**Reply by suricactus** [reply]
@BTreeTech see https://github.com/opengisch/QField/issues/1436

**Reply by BTreeTech** [reply]
from reading that discussion it seems the answer is you can only use the search/ completer for single values at present
